exp/api/v1alpha3/machinepool_webhook.go

```diff
@@ -46,6 +47,14 @@ func (m *MachinePool) Default() {
 	}
 	m.Labels[clusterv1.ClusterLabelName] = m.Spec.ClusterName
 
+	if m.Spec.Replicas == nil {
+		m.Spec.Replicas = pointer.Int32Ptr(1)
+	}
+
+	if m.Spec.MinReadySeconds == nil {
```

Comment:
It might be good to consider changing this to a non-pointer type as part of graduation, since it would eliminate the need to have this defaulting and have the same semantics.

Reply:
this was taken directly from MachineDeployment... I think there is a case for distinguishing user set 0 vs no user value if we decided to change the default to a non-zero value in the future.